numerous/engine/model/model.py

```diff
@@ -379,6 +379,18 @@ def __get_mapping__variable(variable):
                 self.path_variables.update({path: variable.value})  # is this used at all?
 
         self.inverse_aliases = {v: k for k, v in self.aliases.items()}
+        inverse_logged_aliases = {} # {id: [alias1, alias2...], ...}
+        for k,v in self.logged_aliases.items():
+            inverse_logged_aliases[v] = inverse_logged_aliases.get(v, []) + [k]
+
+        self.inverse_logged_aliases = inverse_logged_aliases
+
+        for varname, ix in self.vars_ordered_values.items(): # now it's a dict...
+            var = self.scope_variables[varname]
+            if var.logger_level.value >= self.logger_level.value:
+                if varname in self.inverse_logged_aliases:
+                    for vv in self.inverse_logged_aliases[varname]:
+                        self.logged_variables.update({vv: ix})
```

Comment:
I think we need a test that content of  self.logged_variables is what we expected.